x/tokenomics/keeper/settle_pending_claims.go

```diff
@@ -305,15 +335,118 @@ func (k Keeper) proofRequirementForClaim(ctx sdk.Context, claim *prooftypes.Clai
 	}
 
 	logger.Info(fmt.Sprintf(
-		"claim does not require proof due to compute units (%d) being less than the threshold (%d) and random sample (%.2f) being greater than probability (%.2f)",
-		numClaimComputeUnits,
+		"claim does not require proof due to claimed amount (%s) being less than the threshold (%s) and random sample (%.2f) being greater than probability (%.2f)",
+		claimeduPOKT,
 		proofParams.GetProofRequirementThreshold(),
 		proofRequirementSampleValue,
 		proofParams.GetProofRequestProbability(),
 	))
 	return requirementReason, nil
 }
 
+// slashSupplierStake slashes the stake of a supplier and mints the total slashing
+// amount to the tokenomics module account.
+func (k Keeper) slashSupplierStake(
+	ctx sdk.Context,
+	supplierOperatorAddress string,
+	slashingCount uint64,
+) error {
+	logger := k.logger.With("method", "slashSupplierStake")
+
+	proofParams := k.proofKeeper.GetParams(ctx)
+	slashingPenaltyPerExpiredClaim := proofParams.GetProofMissingPenalty()
+
+	totalSlashingAmt := slashingPenaltyPerExpiredClaim.Amount.Mul(math.NewIntFromUint64(slashingCount))
+	totalSlashingCoin := sdk.NewCoin(volatile.DenomuPOKT, totalSlashingAmt)
+
+	supplierToSlash, supplierFound := k.supplierKeeper.GetSupplier(ctx, supplierOperatorAddress)
+	if !supplierFound {
+		return tokenomicstypes.ErrTokenomicsSupplierNotFound
+	}
+
+	slashedSupplierInitialStakeCoin := supplierToSlash.GetStake()
+
+	var remainingStakeCoin sdk.Coin
+	if slashedSupplierInitialStakeCoin.IsGTE(totalSlashingCoin) {
+		remainingStakeCoin = slashedSupplierInitialStakeCoin.Sub(totalSlashingCoin)
+	} else {
+		logger.Warn(fmt.Sprintf(
+			"total slashing amount (%s) is greater than supplier %q stake (%s)",
+			totalSlashingCoin,
+			supplierOperatorAddress,
+			supplierToSlash.GetStake(),
+		))
+
+		// Set the remaining stake to 0 if the slashing amount is greater than the stake.
+		remainingStakeCoin = sdk.NewCoin(volatile.DenomuPOKT, math.NewInt(0))
+		// Total slashing amount is the whole supplier's stake.
+		totalSlashingCoin = sdk.NewCoin(volatile.DenomuPOKT, slashedSupplierInitialStakeCoin.Amount)
+	}
+
+	// Since staking mints tokens to the supplier module account, to have a correct
+	// accounting, the slashing amount needs to be sent from the supplier module
+	// account to the tokenomics module account.
+	if err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, suppliertypes.ModuleName, tokenomicstypes.ModuleName, sdk.NewCoins(totalSlashingCoin)); err != nil {
+		return err
+	}
+
+	supplierToSlash.Stake = &remainingStakeCoin
+
+	logger.Warn(fmt.Sprintf(
```

Comment:
This is not a warning. 

```suggestion
	logger.Info(fmt.Sprintf(
```